docs/source/index.rst

```diff
@@ -258,19 +257,14 @@ often a better choice.  If you are a *core developer*, then you should start her
    :hidden:
    :caption: Help & reference
 
-   debugging.rst
+   develop.rst
    changelog.rst
-   cheatsheet.rst
    presentations.rst
-   develop.rst
-   faq.rst
```

Comment:
Any thoughts on moving this content into other docs while we are at it?

Reply:
I would be happy with a FAQ if it was more complete.  As is I think it gives an overly narrow view of the project.  I would label this as "future work" for the project at large.

Reply:
Right. Not suggesting we keep the FAQ.

Was suggesting we make sure the few questions it answered land somewhere else in the docs (e.g. rule of thumb about chunk size for instance). This could be one line in the relevant section for instance (e.g. under chunks in Dask Arrays).

Reply:
Ah I see, yes that does make sense.

Reply:
I suggest that we include a "Best Practices" section in each of the collection sections with information like this.  I am happy to help set this up.  I would like to do it in another PR though.

Reply:
Sure "Best Practices" sounds like a good idea.

Doing it another PR is fine. Could I ask that we hold off dropping FAQ from the listing until this content is otherwise integrated?

Reply:
Sure, I guess that serves as extra motivation for me to write those sections quickly :)